plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/StaticMetastoreConfig.java

```diff
@@ -95,4 +98,17 @@ public boolean isEitherThriftOrHttpMetastore()
         boolean hasThriftMetastore = metastoreUris.stream().anyMatch(uri -> "thrift".equalsIgnoreCase(uri.getScheme()));
         return hasHttpMetastore && hasThriftMetastore;
     }
+
+    public String getCatalogName()
+    {
+        return catalogName;
+    }
+
+    @Config(HIVE_METASTORE_CATALOG_NAME)
```

Comment:
Add documentation about this new config property in `hive.md`.

Also specify in the docs the detail discovered about the default catalog name:

```
HiveMetastore uses "hive" as the catalog
````

Reply:
Gentle reminder about the docs

Reply:
please see, added docs to the `metastores.md`
as other metastore configs like `hive.metastore.uri` is there.